### PR TITLE
Ensure that TextureAssetTask does not invert textures by default

### PR DESCRIPTION
**Pull request by broederj**

### CODE REVIEW

File: src/Misc/assetsManager.ts [hunk 1]
```diff
@@ -509,7 +514,7 @@ export class TextureAssetTask extends AbstractAssetTask implements ITextureAsset
         /**
          * Defines if texture must be inverted on Y axis (default is false)
          */
-        public invertY?: boolean,
+        invertY: boolean = false,
```
Comment (deltakosh) [inline src/Misc/assetsManager.ts]: well it seems this parameter is not used then

File: src/Misc/assetsManager.ts [hunk 2]
```diff
@@ -520,6 +520,8 @@ export class TextureAssetTask extends AbstractAssetTask implements ITextureAsset
          */
         public samplingMode: number = Texture.TRILINEAR_SAMPLINGMODE) {
         super(name);
+
+        this.invertY = invertY;
```
Comment (deltakosh) [inline src/Misc/assetsManager.ts]: so why do you not let the public invertY and just set it up to false by default?

Comment (broederj) [reply src/Misc/assetsManager.ts]: Thanks, yes :D

**Comment by broederj** [comment]
Great! Now let's see how many people will feel this is a breaking change ;-) It could be for some implementations, I guess.

**Comment by deltakosh** [comment]
Does not categorize as breaking changes as it was a bug :)

**Comment by deltakosh** [comment]
Well now that Im thinking more about it, naybe we should set it to true by default and update the doc

So we will not break anything!

I'll do the change

**Pull request merged**